src/components/security_manager/test/ssl_context_test.cc

```diff
@@ -336,38 +366,49 @@ INSTANTIATE_TEST_CASE_P(
                                         security_manager::TLSv1_1,
                                         kFordCipher,
                                         kFordCipher),
+#if OPENSSL_VERSION_NUMBER < CONST_SSL_METHOD_MINIMAL_VERSION
                       ProtocolAndCipher(security_manager::TLSv1,
                                         security_manager::SSLv3,
                                         kFordCipher,
                                         kFordCipher),
+#endif
                       ProtocolAndCipher(security_manager::TLSv1_1,
                                         security_manager::TLSv1,
                                         kFordCipher,
                                         kFordCipher),
+#if OPENSSL_VERSION_NUMBER < CONST_SSL_METHOD_MINIMAL_VERSION
                       ProtocolAndCipher(security_manager::TLSv1_1,
                                         security_manager::SSLv3,
                                         kFordCipher,
                                         kFordCipher),
+#endif
                       ProtocolAndCipher(security_manager::TLSv1_2,
                                         security_manager::TLSv1,
                                         kFordCipher,
                                         kFordCipher),
                       ProtocolAndCipher(security_manager::TLSv1_2,
                                         security_manager::TLSv1_1,
                                         kFordCipher,
-                                        kFordCipher),
+                                        kFordCipher)
+#if OPENSSL_VERSION_NUMBER < CONST_SSL_METHOD_MINIMAL_VERSION
+                          ,
                       ProtocolAndCipher(security_manager::TLSv1_2,
                                         security_manager::SSLv3,
                                         kFordCipher,
                                         kFordCipher),
+
                       ProtocolAndCipher(security_manager::SSLv3,
                                         security_manager::TLSv1,
                                         kFordCipher,
                                         kFordCipher),
+
                       ProtocolAndCipher(security_manager::SSLv3,
                                         security_manager::TLSv1_1,
                                         kFordCipher,
                                         kFordCipher)));
+#else
+                          ));
+#endif
```

Comment:
@ValeriiMalkov Not sure that `#ifdef` is needed

Reply:
@AByzhynar fixed at the commit e60c4f93668e3be86a3d3f9ac5fe1cac3b3b6f78